weave/op_def_type.py

```diff
@@ -187,12 +187,18 @@ class RefJSONEncoder(json.JSONEncoder):
     SPECIAL_REF_TOKEN = "__WEAVE_REF__"
 
     def default(self, o):
+        from .trace_server.graph_client_trace import TraceNounRef
```

Comment:
First time encountering "TraceNounRef" for me. It feels like an odd name (unexpected)

Reply:
fixed